src/tickit/cli.py

```diff
@@ -33,19 +33,24 @@ def main(ctx: Context, log_level: str):
 
 
 @main.command(help="run one or more components without a scheduler")
-@click.argument("components", nargs=-1)
+@click.argument("components", required=True, nargs=-1)
 @click.argument("config_path")
 @click.option("--backend", default="kafka", type=click.Choice(list(interfaces(True))))
-def components(config_path: str, components: Set[ComponentID], backend: str) -> None:
+def components(
+    config_path: str,
+    components: Optional[Union[Set[ComponentID], Literal["all"]]],
+    backend: str,
+) -> None:
     """Runs one or more simulated components from a configuration file.
 
     If you pass it no componentID arguments it will run all of the components in the
     configuration file.
 
     Args:
         config_path (str): The path to the configuration file.
-        components Set[ComponentID]: The name of the components to be run, seperated by
-            whitespace.
+        components (Optional[Union[Set[ComponentID], Literal["all"]]]): The name of the
```

Comment:
Possibly an unlikely situation but what happens if you have a component called 'all'?

Reply:
I did not think of that. Back to blank for all?